gen/github.com/jsonnet-libs/k8s-libsonnet/1.19/_custom/volumeMounts.libsonnet

```diff
@@ -159,6 +173,9 @@ local d = import 'doc-util/main.libsonnet';
     v1+: {
       job+: patch,
     },
+    v1beta1+: {
+      cronJob+: patch,
```

Comment:
is not the same effect to just add the `jobTemplate` here and have no if condition in the patches above ?

Reply:
Not really, because then I would have to also add this to `mapContainers.libsonnet` file that `super.mapContainers` depends on. I tried that earlier and it seemed more hacky.

Also, all other objects expose it at the top-level for the object, not as part of inner objects.

A third option was to expose `withVolumeMountsMixin` (perhaps as `withPodTemplateVolumeMountsMixin`) and `withPodAnnotationMixin` using separate file/files in `libs/k8s/custom/core`, but that would make the change much more complex and I am not sure if codebase would ever use it, so it may not be worth exposing externally.